server/src/main/java/io/littlehorse/common/model/command/subcommand/ReassignedUserTask.java

```diff
@@ -0,0 +1,85 @@
+package io.littlehorse.common.model.command.subcommand;
+
+import com.google.protobuf.Message;
+import io.littlehorse.common.LHConfig;
+import io.littlehorse.common.LHDAO;
+import io.littlehorse.common.model.LHSerializable;
+import io.littlehorse.common.model.command.AbstractResponse;
+import io.littlehorse.common.model.command.SubCommand;
+import io.littlehorse.common.model.objectId.NodeRunId;
+import io.littlehorse.common.model.wfrun.NodeRun;
+import io.littlehorse.common.model.wfrun.UserTaskRun;
+import io.littlehorse.common.proto.ReassignedUserTaskPb;
+import io.littlehorse.sdk.common.exception.LHSerdeError;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ReassignedUserTask extends SubCommand<ReassignedUserTaskPb> {
```

Comment:
Nit Pick: I would personally name this `ReassignUserTask` rather than `ReassignedUserTask`.

Generally, the form of a command is `<verb>-<noun>` such as "wash car", "run workflow" etc. This is `<past participle>-<noun>`, like "washed car", "ran workflow".

For consistency, I'd prefer the other name. @sauljabin what do you think?